README.md

```diff
@@ -1,2 +1,20 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+a - Checks if a list has 2 or more of the same items inside it.
+
+b - Because it would take a lot longer to calculate.
+
+c - Because j only ever needs to check the numbers i has been through.
+
+d - Because j doesn't need to go through the whole list every time.
+
+e - No, it is now linear. As performance will grow linearly and in direct proportion to the size of the input.
+
+f - According to "https://wiki.python.org/moin/TimeComplexity" Python's sort function is O(n log n) in big-O notation.
+
+g - O(n log n) seeing as it is using the sort function.
+
+h - Then 2nd algorithm would run faster as it doesn't need to loop as much thanks to the sorting function.
+
+i - If the list is already sorted than the "slower" algorithm will act faster as it can skip the sorting stage.
```

Comment:
In this case the first algorithm is still O(n^2) but the second is O(n), making the second still faster. One possible answer is if n is small, as then it may be that n log n > n^2.